src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,38 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__event-multiday {
+		background-color: rgba(var(--color-accent-primary), .1);
+		border-radius: 6px;
+		border-style: solid;
+		border-width: 0 2px 0px 2px;
+		border-color: var(--color-accent-primary);
+		padding: 1px var(--spacer-1);
+		display: flex;
+		flex-wrap: wrap;
+		align-items: center;
+		margin-bottom: var(--spacer-0);
+
+		&:hover {
+			background-color: rgba(var(--color-accent-primary), .2);
+		}
+	}
+
+	.tribe-events-calendar-month__event-multiday-featured-icon {
```

Comment:
We've used this style a few times now, and its the exact same style to get the featured icon. I've added svg icon classes in this PR: https://github.com/moderntribe/tribe-common/pull/1029. Let's use these going forward (apply these classes to the `<em>` element).